docs/monitoring/manually-setting-jmx-containers.md

```diff
@@ -0,0 +1,67 @@
+title: Manual set up of JMX monitoring for Java process running in containers
```

Comment:
title: Manual set up of JMX monitoring for Java process running in containers
=>
title: Manually setting JMX monitoring for Java processes running in containers